packages/edit-site/src/components/block-editor/editor-canvas.js

```diff
@@ -25,7 +25,16 @@ import {
 	FOCUSABLE_ENTITIES,
 	NAVIGATION_POST_TYPE,
 } from '../../utils/constants';
-import { computeIFrameScale } from '../../utils/math';
+
+const FRAME_SIZE = 20;
+const ZOOM_OUT_MAX_WIDTH = 800;
```

Comment:
Why is there a max width?

Reply:
Add extra padding when the window is really wide.